virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc

```diff
@@ -8,10 +8,6 @@ toc::[]
 You can configure a Single Root I/O Virtualization (SR-IOV) device for virtual machines in your cluster.
 This process is similar but not identical to configuring an SR-IOV device for {product-title}.
 
-[NOTE]
-====
-Live migration is supported for virtual machines that are attached to an SR-IOV network interface only if the `sriovLiveMigration` feature gate is enabled in the HyperConverged Cluster custom resource (CR). When the `spec.featureGates.sriovLiveMigration` field is set to `true`, the `virt-launcher` pod runs with the `SYS_RESOURCE` capability. This might degrade its security.
```

Comment:
````virt-launcher` pod runs with the `SYS_RESOURCE` capability. This might degrade its security.```

This is still true, right? @ormergi

Reply:
@myakove Nope, we dropped SYS_RESOURCE cap since https://github.com/kubevirt/kubevirt/pull/5558 thanks to https://github.com/kubevirt/kubevirt/pull/5328 :slightly_smiling_face: